native/packages/react-native-bpk-component-nudger/src/BpkNudger-test.common.js

```diff
@@ -128,12 +129,13 @@ const commonTests = () => {
               value={3}
               decreaseButtonLabel="Decrease"
               increaseButtonLabel="Increase"
-              onChange={onPressFn}
+              onChange={() => null}
             />
           </BpkThemeProvider>,
         )
         .toJSON();
       expect(tree).toMatchSnapshot();
+      expect(consoleErrorFn.mock.calls.length).toBe(1);
```

Comment:
Why are we expecting this?

Reply:
yes we're expecting an error from the theme provider if the theming object does not contain the appropriate theming properties